templates/template.html

```diff
@@ -34,7 +34,17 @@
   margin-bottom: 1rem;
 }
     </style>
-
+<script type="text/javascript">
+    $(function (){
+        $("#search_btn").click(function (){
+            if ($("#search_text").val() == '') {
```

Comment:
@HeShuo-dot What if the search text is a string in which all 3 characters are whitespace?
                        Why do not you use the "required" attribute for the input box?

-Hui